app/scripts/components/common/blocks/block-map.tsx

```diff
@@ -1,8 +1,7 @@
 import React, { useMemo, useState, useEffect } from 'react';
 import styled from 'styled-components';
-import { DatasetDatumFnResolverBag, ProjectionOptions, datasets } from 'veda';
+import { ProjectionOptions } from 'veda';
```

Comment:
Yes! I was thinking about how we could replace this type import soon. Since we will need to replace most of type imports once Oliver's branch is merged, I don't think we need to make it as a dependency now. but a note can help